cfn-resources/cloud-backup-snapshot/README.md

```diff
@@ -0,0 +1,64 @@
+# MongoDB::Atlas::BackupSnapshot
+
+## Description
+Returns, adds,  and removes cloud backup.
```

Comment:
```suggestion
Returns, takes, edits, and removes Cloud Backup snapshots
```
please follow Open API doc definitions https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Cloud-Backups

Reply:
This issue is fixed